knack/testsdk/base.py

```diff
@@ -117,7 +117,7 @@ def setUp(self):
 
         # set up cassette
         cm = self.vcr.use_cassette(self.recording_file)
-        self.cassette = cm.__enter__()
+        self.cassette = cm.__enter__()  # pylint: disable=unnecessary-dunder-call
```

Comment:
`cm.__exit__` is not called immediately in this method to release `self.cassette`, but registered in `self.addCleanup`, so this dunder call is no unnecessary.